packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -281,11 +282,7 @@ export const makeZCFZygote = async (
     ['canBeUpgraded', 'canUpgrade'].includes(meta.upgradability);
 
   /** @type {ZCF} */
-  // Using Remotable rather than Far because there are too many complications
-  // imposing checking wrappers: makeInvitation() and setJig() want to
-  // accept raw functions. assert cannot be a valid passable! (It's a function
-  // and has members.)
-  const zcf = Remotable('Alleged: zcf', undefined, {
+  const zcf = prepareExo(zcfBaggage, 'zcf', ZcfI, {
```

Comment:
happy to see what a clean change this was to Exo. Since it's a singleton there's no state shape or need to work about migration. https://github.com/Agoric/agoric-sdk/issues/7407